Microsoft.Research/ContractAdornments/Extension/ContractAdornments.csproj

```diff
@@ -423,13 +423,8 @@
     <CodeAnalysisIgnoreBuiltInRules>false</CodeAnalysisIgnoreBuiltInRules>
     <CodeAnalysisFailOnMissingRules>false</CodeAnalysisFailOnMissingRules>
   </PropertyGroup>
-  <PropertyGroup>
```

Comment:
:memo: The changes to this file should actually have been included in #66 but I forgot this change.